light_node.cpp

```diff
@@ -107,7 +107,7 @@ void LightNode::setManufacturerCode(uint16_t code)
         case VENDOR_UBISYS:  name = QLatin1String("ubisys"); break;
         case VENDOR_BUSCH_JAEGER:  name = QLatin1String("Busch-Jaeger"); break;
         //case VENDOR_EMBER:   // fall through
-        case VENDOR_HEIMAN:  name = QLatin1String("Heiman"); break;
+        //case VENDOR_HEIMAN:  name = QLatin1String("Heiman"); break;
```

Comment:
Not too sure if this has the potential to break anything...

Reply:
No, I don't think, I haven't see in code some check with "Heiman"

Reply:
I can only tell for Phoscon App, currently we have the following "Heiman" matching:

```json
	"Heiman": [
		{ "modelid": "SMOK_" },
		{ "modelid": "Smoke" },
		{ "modelid": "WarningDevice" },
		{ "modelid": "WATER_" },
		{ "modelid": "Water" },
		{ "modelid": "CO_" },
		{ "modelid": "COSensor" },
		{ "modelid": "DOOR_"},
		{ "modelid": "Door" },
		{ "modelid": "PIR_"},
		{ "modelid": "TH-" },
		{ "modelid": "GAS"},
		{ "modelid": "SmartPlug"},
		{ "modelid": "SKHMP30"}
	]
```

From the `modelid` they appear to be real Heiman devices not Tuya.

Reply:
Not sure about the last entry, but the others are genuine Heiman devices, if memory serves.